go/kbfs/test/engine_common.go

```diff
@@ -42,8 +42,8 @@ func setBlockSizes(t testing.TB, config libkbfs.Config, blockSize, blockChangeSi
 
 func maybeSetBw(t testing.TB, config libkbfs.Config, bwKBps int) {
 	if bwKBps > 0 {
-		config.SetBlockOps(libkbfs.NewBlockOpsConstrained(
-			config.BlockOps(), bwKBps))
+		// TODO: reinstate constrained bandwidth tests.
```

Comment:
Was this breaking as a result of the lint changes or something? I'm confused.

Reply:
Turns out this broke a long time ago (in the days of the old `kbfs` repo) on a refactor of the `BlockOps` functions.  But these constrained benchmark tests are either run manually (and probably haven't even been run since that happened) or only used as a crutch to set the background flushes flag, so it didn't seem worth it to try to fix as part of this PR.